openedx/core/djangoapps/safe_sessions/middleware.py

```diff
@@ -317,13 +317,18 @@ def process_response(self, request, response):
 
         if not _is_cookie_marked_for_deletion(request) and _is_cookie_present(response):
             try:
+                is_from_logout = _is_from_logout(request)
```

Comment:
Ah. Thanks. I now see there were 2 logs within the same `if` statement.  Thanks.